pkg/middleend/qos_test.go

```diff
@@ -947,20 +947,20 @@ func TestMiddleEnd_GetQosVolume(t *testing.T) {
 	}
 }
 
-func TestMiddleEnd_QosVolumeStats(t *testing.T) {
+func TestMiddleEnd_StatsQosVolume(t *testing.T) {
 	tests := map[string]struct {
 		in      *_go.ObjectKey
```

Comment:
`in` can be `string` since we need to pass just name and `QosVolume` does not have `ObjectKey`

Reply:
replaced